recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
+
+build:
+  noarch_python: True
+  number: 0
+# since this is noarch, only build on linux
+  skip: True  # [not linux]
```

Comment:
Assuming I can trust Github's search, it doesn't look like any packages on conda-forge are built this way. Instead it appears like every package builds a specific version for each platform. I don't know what the actual conda-forge policy on this is though.